src/components/Recollect/index.js

```diff
@@ -17,6 +17,8 @@ class Recollect extends Component {
       options,
     );
     let loader = new window.Recollect.Widget.Loader(widgetOptions);
+    const scrollToId = window.location.hash.split('#')[1]
+    document.getElementById(scrollToId).scrollIntoView(true);
```

Comment:
..same here for auto scrolling when a link is clicked to this location. Copy/paste a link into a new tab to see the effect!